tests/iscsi/iscsi_test.py

```diff
@@ -0,0 +1,121 @@
+"""
```

Comment:
please correct the module comments according to iSCSI

Reply:
Done.